### DISCUSSION TITLE
Container analysis

### DISCUSSION THREAD

**Discussion by secsonthebeach** [e-m-b-a/emba#635] [Ideas]
Hello, is emba capable of, or are there plans for analyzing container images?

**Comment by m-1-k-3** [comment]
you should be able to start EMBA with `-C <CONTAINER-ID>` and EMBA should do the export and testing for you 

![image](https://github.com/e-m-b-a/emba/assets/497520/7a412820-6f0c-48b2-a5c8-5fba7df009e2)
![image](https://github.com/e-m-b-a/emba/assets/497520/ad84d21b-108b-43f4-a2fe-088b20b23df0)

**Reply by secsonthebeach** [reply]
okay great! I should have caught that in the documents. Thank you for your quick reply!
